samples/matter/light_bulb/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		zephyr,console = &uart30;
+		zephyr,shell-uart = &uart30;
+	};
+
+	aliases {
+		factory-data = &factory_data;
+		factory-data-memory-region = &rram0;
+	};
+};
+
+/* Matter needs at least 700 kB Flash size, instead of default 344 kB. */
```

Comment:
NIT: This is a bit obsolete comment, and it can be removed or changed somehow because we use the full 1.5 MB now.

Reply:
Done